media/Config/exportConfig.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var importConfigPath = '';
+
+// Get file name from the path.
```

Comment:
Maybe add an example will help understading
```
filepath = '/path/to/filename.ext';
result = getFileName(filepath);
// result will contain 'filename'
```

Reply:
You mean add a comment like this?
```
/**
 * Get file name from the file path.
 * filePath = '/path/to/filename.ext';
 * result = getFileName(oneToolList);
 * result will contain 'filename'.
 */
```

Reply:
> You mean add a comment like this?

Yup :)